src/test/java/jnr/posix/FileStatTest.java

```diff
@@ -157,4 +155,12 @@ public void filestatFailed() throws Throwable {
             }
         }
     }
+
+    @Test public void statIsWritable() throws Throwable {
+      String tempDirectory = System.getenv("TEMP");
+      FileStat st = posix.stat(tempDirectory);
+
+      // FIXME: This fails on JRuby, but succeeds here. Why?
```

Comment:
Having not really looked at this at all, Ruby applies extra checks around the permissions of the `$ENV['TMPDIR']` directory to prevent security issues. One is checking that the directory isn't world writable, I believe.